nn-git-log/README.md

```diff
@@ -0,0 +1,113 @@
+# Git Log
+
+Now that you know how to do the basics in git and started working on the repo, a few commits in and you might ask yourslef, how do we visualize these commits and changes we have made in the repository?  
```

Comment:
- **Now that you know the basics of Git and have started working on the repo, you might ask...**
- yourslef - yourself